crates/block-producer/src/subcommand/export_block.rs

```diff
@@ -0,0 +1,158 @@
+use std::fs;
```

Comment:
We should create a new `crates/cli` crate, and move cli related files such as `block-producer/src/main.rs` and `export_block.rs` to the new crate.

Reply:
May worth clarifying: what's the difference between `cli` and `tools`? What about `replay-chain`?

Reply:
`crates/godwoken-bin` crate ?

Reply:
Now extract code to `godwoken-bin` crate